cmake/init_target.cmake

```diff
@@ -44,6 +44,7 @@ function(init_target target_name) # init_target(my_target folder_name)
             HAVE_SSE2
         )
 
+        if (MSVC) # implies not MINGW
```

Comment:
what about clang-cl? also, this makes indentation wrong

Reply:
I can try to add some CI build workflow if required (need some time).